browser/ui/BUILD.gn

```diff
@@ -235,6 +235,7 @@ source_set("ui") {
     "//brave/browser/gcm_driver",
     "//brave/browser/profiles",
     "//brave/browser/tor",
+    "//components/dom_distiller/content/browser",
```

Comment:
add this only if speedreader buildflag is on

Reply:
fixed